frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -95,7 +95,7 @@ class AccountsSummary extends Component<Props, State> {
     }
 
     public componentWillUnmount() {
-        window.clearInterval(this.summaryReqTimerID);
```

Comment:
Good change.

> It's worth noting that the pool of IDs used by setTimeout() and setInterval() are shared, which means you can technically use clearTimeout() and clearInterval() interchangeably. However, for clarity, you should avoid doing so.

(from https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/clearTimeout)

Although "pool of IDs used by setTimeout() and setInterval() are shared" may depend on the browser, I believe.